src/sentry/static/sentry/app/data/forms/sentryApplication.jsx

```diff
@@ -0,0 +1,41 @@
+const forms = [
+  {
+    // Form "section"/"panel"
+    title: 'Application Details',
+    fields: [
+      {
+        name: 'name',
+        type: 'string',
+        required: true,
+        placeholder: 'e.g. My Application',
+        // additional data/props that is related to rendering of form field rather than data
+        label: 'Name',
+        help: 'Human readable name of your application.',
+      },
+      {
+        name: 'webhookUrl',
+        type: 'string',
+        required: true,
+        label: 'Webhook URL',
+        placeholder: 'e.g. https://example.com/sentry/webhook/',
+        help: 'The URL Sentry will make requests to on installation changes.',
+      },
+      {
+        name: 'redirectUrl',
+        type: 'string',
+        label: 'Redirect URL',
+        placeholder: 'e.g. https://example.com/sentry/setup/',
+        help: 'The URL Sentry will redirect users after installation.',
+      },
+      {
+        name: 'overview',
+        type: 'textarea',
+        label: 'Overview',
+        placeholder: 'Put whatever you want here...',
```

Comment:
Do use use ellipsis often? `\u2026` is the code.

Reply:
Im not sure we even need a placeholder tbh - @markstory  thoughts on removing it?